localization/english/opinions/wc_general_opinions_l_english.yml

```diff
@@ -1,24 +1,24 @@
 ﻿l_english:
- being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- being_undead_intolerant:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- 
- loyal_undead:0 "Loyal [GetTrait('being_undead').GetName( GetNullCharacter )]"
- attempted_to_kill_me:0 "Attempted to Kill Me"
- 
- being_demon_crime:0 "[GetTrait('being_demon').GetName( GetNullCharacter )]"
- being_demon_intolerant:0 "[GetTrait(being_demon).GetName( GetNullCharacter )]"
- 
- being_void_crime:0 "[GetTrait('being_void').GetName( GetNullCharacter )]"
- being_void_intolerant:0 "[GetTrait(being_void).GetName( GetNullCharacter )]"
- 
- being_life_crime:0 "[GetTrait('being_life').GetName( GetNullCharacter )]"
- being_life_intolerant:0 "[GetTrait(being_life).GetName( GetNullCharacter )]"
- 
- being_order_crime:0 "[GetTrait('being_order').GetName( GetNullCharacter )]"
- being_order_intolerant:0 "[GetTrait(being_order).GetName( GetNullCharacter )]"
- 
- being_light_crime:0 "[GetTrait('being_light').GetName( GetNullCharacter )]"
- being_light_intolerant:0 "[GetTrait(being_light).GetName( GetNullCharacter )]"
+  being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )|V]"
```

Comment:
Noticed one weird thing. There are 2 spaces at the beginning of the lines in this file. It's not a bug but it's an extra change left from the merge.

Reply:
Removed the 1 of the spaces.